File: [프로그래머스] 연습문제_크기가 작은 부분 문자열.py
def solution(t, p):
    answer, num, i = 0, 0, 0
    while True:
        if i+len(p) > len(t):   #out of index 방지
            break
        num = int(t[i:i+len(p)])    #p의 길이만큼 문자열 슬라이싱
        if num <= int(p):
            answer += 1
        i += 1
    return answer

File: [프로그래머스] 연습문제_크기가 작은 부분 문자열.js
function solution(t, p) {
    var answer = 0;
    let pl = p.length;
    let str = ''
    
    for(let i=0; i <= t.length-pl; i++){
        str = t.slice(i, i+pl);
        if (parseInt(str) <= parseInt(p)){
            answer++;
        }
    }
    return answer;
}